packages/grafana-ui/src/components/Forms/RadioButtonList/RadioButtonList.tsx

```diff
@@ -48,7 +48,7 @@ export function RadioButtonList<T>({
 
         return (
           <RadioButtonDot
-            key={itemId}
+            key={`${option.label}-${index}`}
```

Comment:
I think, if we're going to include `index`, then it that's all the key should be. Including `index` _at all_ in the key is the thing that kind of breaks it, so `option.label` doesn't really do anything for us here.

Reply:
Was thinking the same, it doesn't look like the components in the list will be dynamically removed or added and the key should be unique only in the scope of the parent component, so `index` would probably work here.

Reply:
yes, agreed, that does not change anything actually